ansible_collections/arista/avd/docs/contribution/development-tooling.md

```diff
@@ -0,0 +1,238 @@
+Before you can leverage the VSCode Dev Container ensure to have the following tools installed on your workstation:
+
+- [Docker](https://docs.docker.com/engine/install/).
+- [Visual Studio Code](https://code.visualstudio.com/) and the [Dev Container](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension.
+
+The AVD repository contains a devcontainer definition at the root of the project.
+This will be executed automatically by VSCode by following these steps:
+
+1. Start VSCode on your worksation
+2. Select "File/Open Folder..." and open the cloned avd GitHub repository (your fork).
+3. A pop up will appear with "Folder contains a Dev Container configuration file. Reopen folder to develop in a container." Select "Reopen in Container".
+4. VScode will re-open and start in the AVD Dev Container.
+
+!!! Warning
+    Note, at this time the AVD Dev Container doesn't support docker-in-docker, therefore some of the tests that rely on Docker can not be performed locally. Please rely on the CI pipeline results and logs for troubleshooting.
+
+### Local Python environments
+
+Developing with your local Python environment requires you to configure and install the AVD project development tools and dependencies installed on your workstation:
+
+- [Python 3.10](https://docs.python.org/) or later and [Python Virtual Environments](https://docs.python.org/3/tutorial/venv.html).
+- Additional AVD Python package dependencies.
+- [Make](https://www.gnu.org/software/make/manual/make.html): Leveraged for automating software building and test procedures.
+- [Docker](https://docs.docker.com/engine/install/) (Optional): Some of the tests require docker to be executed locally and useful when troubleshooting failures of the CI pipeline.
+
+Recommended steps with Python virtual environment:
+
+1. Create and activate a Python virtual environment.
+2. Install Python requirements located in the AVD repository: [requirements-dev.txt](https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/requirements-dev.txt) and [requirements.txt](https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/requirements.txt).
+
+!!! note
+    Ensure the virtual environment is located outside of the AVD project directory.
+
+```shell
+# Create a Python virtual environment `python -m venv <virtual-environment-name>`.
+python3 -m venv avd-venv
+
+# Activate Python virtual environment `source <virtual-environment-name>/bin/activate`.
+source avd-venv/bin/activate
+
+# Install AVD project requirements-dev.txt and requirements.txt in your Python Virtual environment.
+# Requirements files are located in `ansible_collections/arista/avd` of the avd repository.
+pip3 install -r ansible_collections/arista/avd/requirements-dev.txt -r ansible_collections/arista/avd/requirements.txt --upgrade
```

Comment:
Steps on lines 66 and 69 assumes we are in the 'root' folder with subfolders 'avd' and 'avd-venv'

```suggestion
pip3 install -r avd/ansible_collections/arista/avd/requirements-dev.txt -r avd/ansible_collections/arista/avd/requirements.txt --upgrade
```